### PR TITLE
unique_all

### PR DESCRIPTION
**Pull request by Archymade**

### CODE REVIEW

**Comment by sherry30** [comment]
> I keep getting this: `` ivy/functional/ivy/set.py:16 in public function `unique_all`: D207: Docstring is under-indented `` for the numpy-docstrings-test. Any ideas?

The Example part of the docstring was under-indented in the unique_all function. Under indentation can be fixed by selecting the part and pressing shift+tab once to unindent it then press tab to indent it properly.

**Comment by Archymade** [comment]
I still can't figure out why the nightly tests are still failing, but I'll still work on it today.

File: ivy/functional/ivy/set.py [hunk 1]
```diff
@@ -74,7 +74,34 @@ def unique_all(
         .. note::
            The order of unique elements is not specified and may vary between
            implementations.
-
+   
+    Examples
```
Comment (juliagsy) [inline ivy/functional/ivy/set.py:78]: please format the examples into different sections, which are functional and instance, with their respective subsections if applicable. Docstrings example samples are shown here (https://lets-unify.ai/ivy/deep_dive/11_docstring_examples.html) and there are multiple points listed down for reference, thanks!

**Review by juliagsy — CHANGES_REQUESTED** [review]
comments by section are the feedbacks, thanks!

**Comment by juliagsy** [comment]
Hey there! As the PR is inactive for more than 7 days, it will be closed in 24 hours if there are no further response! Thank you!

**Comment by Archymade** [comment]
Tests that were passing previously are failing now. I can't see why, as I haven't changed anything other than the docstring. Please review.

**Comment by juliagsy** [comment]
Hey there! Thanks for contributing! The failing tests which previously passed are mainly caused by the master branch so you may leave that for now! I'll review them as soon as possible (by today), then request changes or merge if there isn't further issues! Thanks!

**Comment by Archymade** [comment]
Got it. Thanks.

**Comment by Archymade** [comment]
Thanks for the merge! I have already worked on the `unique_values` and `unique_inverse` functions. Do I commit and push those too?

**Comment by juliagsy** [comment]
Hello! As this PR is merged, you'll have to open a new PR and request for review! Then, we'll decide if further changes are needed or it's good for a merge! Thank you!

**Pull request merged**